bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/regexp/TreeTraverser.java

```diff
@@ -747,11 +747,11 @@ private Token readTokenInFlagsExpression() {
      * <p>
      * <code>
      * ReEscape :=
-     *    NumericEscape
-     *    | ControlEscape
```

Comment:
This formatting changes are initiated by IntelliJ setting in the local machine.
Imo, We need to remove these